katello.gemspec

```diff
@@ -59,6 +59,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "pulp_deb_client", ">= 2.10.0", "< 2.12.0"
   gem.add_dependency "pulp_rpm_client", ">=3.10.0", "< 3.11.0"
   gem.add_dependency "pulp_certguard_client", "< 2.0"
+  gem.add_dependency "pulp_python_client", ">= 3.2.0"
```

Comment:
We'll need to either wait to add this or get `pulp_python_client` into foreman-packaging before merging.

Reply:
Related: https://github.com/theforeman/foreman-packaging/pull/6790

Reply:
@ianballou if that gets merged into foreman, would it make sense to remove this? https://github.com/Katello/katello/blob/89830530de2f46ab6c88f99807b1fe54efa1901a/lib/katello/repository_types/python.rb#L1

Reply:
That PR only has to do with installing production versions of Katello.  You'll still need to require the client somewhere in the code.